kotskinds/apis/kots/v1beta1/config_types.go

```diff
@@ -56,12 +56,22 @@ type ConfigItem struct {
 	CountByGroup  map[string]int         `json:"countByGroup,omitempty"`
 	Templates     []RepeatTemplate       `json:"templates,omitempty"`
 	ValuesByGroup ValuesByGroup          `json:"valuesByGroup,omitempty"`
+	Validation    *ConfigItemValidator   `json:"validation,omitempty"`
```

Comment:
Do we want to consider allowing multiple validators per config item?  What if we add more types in the future and a vendor wants to use more that one?  Maybe it can all be accomplished with a sufficiently complex regex, but just a thought...

Reply:
Yes